elearning-backend/app/Http/Controllers/API/SessionsController.php

```diff
@@ -3,20 +3,18 @@
 namespace App\Http\Controllers\API;
 
 use App\Http\Controllers\Controller;
-use App\Http\Requests\UserFieldPostRequest;
+use App\Http\Requests\UserLoginPostRequest;
 use App\Models\User;
-use Illuminate\Http\Request;
 
 class SessionsController extends Controller
 {
-    public function login(Request $request)
+    public function login(UserLoginPostRequest $request)
     {
-        $validator = UserFieldPostRequest::createFrom($request);
-        $validator->module = "login";
+        $validator = $request->validated();
```

Comment:
```suggestion
```
You can now remove this line, since it was already validated in UserLoginPostRequest.
Then use $request from now on

Also apply this to other methods

Reply:
Code has been cleaned up @chevyB